rdr_service/tools/tool_libs/resource_tool.py

```diff
@@ -383,7 +499,7 @@ def run():
                                 action="store_true")  # noqa
 
 
-    genomic_parser = subparser.add_parser("genomic")
+    genomic_parser = subparser.add_parser("genomic", )
```

Comment:
I am not sure what the effect of this change is;  just checking that it isn't an inadvertent edit?

Reply:
That comma shouldn't be there, I'll remove it.